### DISCUSSION TITLE
Swiping Comparison

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1283] [Q&A] [answered]
when Comparison Type == RealityData , the vp can spit left show  RealityData  and right not show RealityData  ,
but how can  the vp can spit left show  [model] and right not show [model]?

in the RenderMode , 
export declare enum RenderMode { 
    Wireframe = 0, 
    SmoothShade = 6, 
    HiddenLine = 3, 
    SolidFill = 4
}

**Comment by williamkbentley** [comment, marked as answer]
Would a view clip plane fit your needs. Try the [View Clipping Sample](https://www.imodeljs.org/sample-showcase/?group=Viewer+Features&sample=view-clip-sample&imodel=Retail+Building+Sample) with an X, Y, Z, or _any_ plane.
